Makefile

```diff
@@ -17,6 +17,9 @@ behatappend:
 behatv:
 	docker-compose run --rm tests bash -c "vendor/bin/behat --config=features/behat.yml --strict --stop-on-failure -v"
 
+behatwip:
+	docker-compose run --rm tests bash -c "vendor/bin/behat --config=features/behat.yml --strict --stop-on-failure --tags '@wip'"
```

Comment:
I don't see the `wip` tag referenced. Is this no longer needed, or something I missed?

Reply:
It's for temporarily adding to a Behat Feature or Scenario while working on it, so it's easier to run just that test during development. For example:
```gherkin
Feature: Does something awesome

  @wip
  Scenario: Handles basic awesomeness requirements
    ... 
```

It's not intended to be committed into the code, which is why you don't see that Work-In-Progress tag (`@wip`) anywhere else.